controller/config/config.go

```diff
@@ -65,9 +77,9 @@ type Api struct {
 }
 
 type Config struct {
-	Enabled            bool
-	Api                Api
-	Enrollment         Enrollment
```

Comment:
I'm not sure why these are showing as changed, even though it's just a formatting change, they don't show as being changed on my local files.

Reply:
It is a white space change, just shifted them all to the left. Should be fine.